doc/history.md

```diff
@@ -71,9 +71,11 @@ and several projects are hosted there including:
 
 In Mai 2019 [Julien Delpangue announced via Twitter](https://twitter.com/juldelplanque/status/1132670416852537344) another fork - stating he wanted to have a clean version for an SQLite3 driver for Pharo. [Julien](https://github.com/juliendelplanque) wanted the code to be independent from UDBC and Glorp. He did not know that one could load only the driver independent from any other - and like Guille he did not contact the original maintainers to cooperate and sort out issues.  
```

Comment:
`He did not know that one could load only the driver independent from any other [...]`

I do not agree with this part of the story because as well as I remember, I had to cut some dependencies in order to be able to load the SQL part only. If I am not wrong the dependencies were in the unit tests. Some unit tests concerning SQLite3 were bound to Glorp in some way.

I'd like to discuss what we can do so this sentence makes me look less as the stupid bad guy. :-)

By the way, I think it is not necessary to associate me to Guille in this part of the document as I did not sync with him to do that... And I have no relationship with DBXTalk/Garage project.

It was for sure awkward what I did: not contacting you to cooperate and the tweet was super rude for all original developers of the project. I admit it and we can keep the link to the tweet this is perfectly ok to me. But the end of this sentence makes me feel that you think I planned to do it in order to harm your project. I can ensure you this is not the case and I am really sorry if you felt it that way...

Reply:
This part of the story is not wrong, especially if we look at the situation and code:

1. There was never a dependency on Glorp for the SQlite3 Library, neither for the base - nor for the tests
    Easily proven by looking at the baseline:

       https://github.com/astares/Pharo-UDBC/blob/master/src/BaselineOfUDBC/BaselineOfUDBC.class.st

   You can additionally check yourself if you follow the load expression that was given on 

      https://github.com/astares/Pharo-UDBC

    where "SQlite3" loads no GLORP packages. So there was no dependency to be cut, no necessity for the fork.
 
2. There were a few test selectors with just "Glorp" in the name - making you think there is a dependency
    on Glorp - but you did not really check and also did not consider to ask. 

    As Pierce explained to you in his mail from 28.05.2019 - this was by intention and without coupling.

```
> #testGlorpAutoID
> #testGlorpUpdate
> #testGlorpUpdateTwice
> #testGlorpCompositeKey
> #testGlorpDoubleInsertPrimaryKey

Glorp itself tests doubly inserting primary keys, updating the same
record twice, etc. I wrote these tests in the SQLite test suite to
isolate what Glorp was testing. When these SQLite tests pass, then
Gliorp's corresponding tests when running over SQLite are also sure to
pass. That's the only reason I named these tests testGlorpXXX.
```

Both things we could have easily answered if you would have contacted Pierce or me. 

I had no problem with the tweet or the fork itself - maybe a little bit with the personalized copyright statement ignoring previous authors. More with the fact that more forks confuse people even more. 

If it is better for you we can change the wording.